index.bs

```diff
@@ -681,14 +706,551 @@ id=] must be the same as the [=window handle=].
+
+      <dt>|value| has a \[[WeakSetData]] [=internal slot=]
+      <dd>Let |remote value| be an [=object=] matching the <code>WeakSetValue</code>
+          production in the [=local end definition=], with the <code>objectId</code>
+          property set to the [=object id for an object=] |value|.
+
+      <dt>|value| has an \[[ErrorData]] [=internal slot=]
+      <dd>Let |remote value| be an [=object=] matching the <code>ErrorValue</code>
+          production in the [=local end definition=], with the <code>objectId</code>
+          property set to the [=object id for an object=] |value|.
+
+      <dt>[=IsPromise=](|value|)
+      <dd>Let |remote value| be an [=object=] matching the <code>PromiseValue</code>
+          production in the [=local end definition=], with the <code>objectId</code>
+          property set to the [=object id for an object=] |value|.
+
+      <dt>|value| has a \[[TypedArrayName]] [=internal slot=]
+      <dd>Let |remote value| be an [=object=] matching the <code>TypedArrayValue</code>
+          production in the [=local end definition=], with the <code>objectId</code>
+          property set to the [=object id for an object=] |value|.
+
+      <dt>|value| has an \[[ArrayBufferData]] [=internal slot=]
+      <dd>Let |remote value| be an [=object=] matching the <code>ArrayBufferValue</code>
+          production in the [=local end definition=], with the <code>objectId</code>
+          property set to the [=object id for an object=] |value|.
+
+      <dt>|value| is a [=platform object=] that implements [=Node=]
+      <dd>
+        1. Let |serialized| be null.
+
+        1. If |node details| is true, run the following steps:
+
+            1. Let |serialized| be a new [=Object=].
+
+            1. [=Set=](|serialized|, "nodeType", [=Get=](|value|, "nodeType"), false)
+
+            1. [=Set=](|serialized|, "nodeName", [=Get=](|value|, "nodeName"), false)
+
+            1. [=Set=](|serialized|, "localName", [=Get=](|value|, "localName"), false)
+
+            1. [=Set=](|serialized|, "nodeValue", [=Get=](|value|, "nodeValue"), false)
+
+            1. Let |child node count| be the size of |serialized|'s [=children=].
+
+            1. [=Set=](|serialized|, "childNodeCount", |child node count|, false)
+
+            1. If |max depth| is null and or equal to 0 let |children| be null.
+               Otherwise, let |children| be an empty list and, for each node
+               |child| in the [=children=] of |value|:
+
+              1. Let |serialized| be the result of [=serialize as a remote value=]
+                 with |child|, |max depth| - 1, |node details| and
+                 |set of known objects|.
+
+              1. Append |serialized| to |children|.
+
+            1. [=Set=](|serialized|, "children", |serialized|, false)
+
+            1. Let |attributes| be a new [=Object=].
+
+            1. For each |attribute| in |value|'s <a spec=dom>attributes</a>:
```

Comment:
```suggestion
            1. For each |attribute| in |value|'s <a>attribute list</a>:
```

This was linked to https://dom.spec.whatwg.org/#concept-attribute, which isn't the right thing, should be https://dom.spec.whatwg.org/#concept-element-attribute.

Not sure about the syntax, maybe `for=Element` is the right fix.